apps/fluent-tester/src/FluentTester/TestComponents/Avatar/consts.ts

```diff
@@ -1,3 +1,16 @@
 export const HOMEPAGE_AVATAR_BUTTON = 'Homepage_Avatar_Button';
 export const AVATAR_TESTPAGE = 'Avatar_TestPage';
 export const JSAVATAR_TESTPAGE = 'Avatar_Win32_TestPage';
+
+export const JSAVATAR_ACCESSIBILITY_LABEL = 'E2E Avatar Accessibility label';
+export const JSAVATAR_ACCESSIBILITY_LABEL_BY_NAME = 'Richard, available';
+export const JSAVATAR_ACCESSIBILITY_HINT = 'E2E Avatar Accessibility hint';
+export const JSAVATAR_ACCESSIBILITY_ROLE = 'none';
+
+export const AVATAR_TEST_COMPONENT = 'AVATAR_TEST_COMPONENT';
+export const AVATAR_SECONDARY_TEST_COMPONENT = 'AVATAR_SECONDARY_TEST_COMPONENT';
```

Comment:
Currently tests are failing and it's because of different test components. If I name it the same, then a couple of tests will pass. I don't know why it's like that. Any help will be appreciated

Reply:
I'm not sure why the tests are failing but I'll continue to look at it.

Reply:
Thank you, Krystal :)

Reply:
I wasn't able to find anything. I'm unable to run E2E tests locally because of some error I can't resolve. I'll try poking at it again tomorrow but you can look at the build artifacts and see if you can find anything there.

If you switch it up and use the methods that are defined in the other component E2E tests (getAccessibilityLabel, getAccessibilityRole, etc.) instead of the added getPrimaryComponentAttribute, does it work then? I don't see why it would help but it's the only thing I can think of trying. Not sure what else is different between your implementation and others.

Reply:
The issue is that the first Avatar is not showing up in the accessibility tree:

![image](https://user-images.githubusercontent.com/4602628/173469314-5f593ddc-2446-48f2-9e29-d7159da77bed.png)

The test framework relies on querying elements and attributes using the accessibility tree. Since the first Avatar is not in the tree, it's failing to find the component, and the tests are failing because all the tests start by trying to find the primary/first element.

Reply:
Unrelatedly the presence badge looks really big on the default size, maybe it should be smaller?

Reply:
> The issue is that the first Avatar is not showing up in the accessibility tree:
> 
> ![image](https://user-images.githubusercontent.com/4602628/173469314-5f593ddc-2446-48f2-9e29-d7159da77bed.png)
> 
> The test framework relies on querying elements and attributes using the accessibility tree. Since the first Avatar is not in the tree, it's failing to find the component, and the tests are failing because all the tests start by trying to find the primary/first element.

Oh, it was absent, because I set accessibilityRole to 'none'. Thank you for finding it